flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ddl/AlterTableDropConstraintOperation.java

```diff
@@ -20,24 +20,42 @@
 
 import org.apache.flink.table.catalog.ObjectIdentifier;
 
+import javax.annotation.Nullable;
+
+import java.util.Optional;
+
 /** Operation of "ALTER TABLE ADD [CONSTRAINT constraintName] ..." clause. * */
 public class AlterTableDropConstraintOperation extends AlterTableOperation {
-    private final String constraintName;
+
+    private final boolean isPrimaryKey;
+    private final @Nullable String constraintName;
 
     public AlterTableDropConstraintOperation(
-            ObjectIdentifier tableIdentifier, String constraintName) {
+            ObjectIdentifier tableIdentifier,
+            boolean isPrimaryKey,
```

Comment:
Will it better to wrap `isPrimaryKey` and `constraintName` to a class may called `Constraint`?

Reply:
I think it is not needed, here only two variable.